modules/core/arrow-core-data/src/main/kotlin/arrow/typeclasses/Monad.kt

```diff
@@ -50,12 +50,6 @@ interface Monad<F> : Selective<F> {
   fun <A> Kind<F, Kind<F, A>>.flatten(): Kind<F, A> =
     flatMap(::identity)
 
-  fun <A, B> Kind<F, A>.followedBy(fb: Kind<F, B>): Kind<F, B> =
-    flatMap { fb }
-
-  fun <A, B> Kind<F, A>.followedByEval(fb: Eval<Kind<F, B>>): Kind<F, B> =
-    flatMap { fb.value() }
-
```

Comment:
Can you overwrite those here? I think a single flatMap is more performant than `Apply.map` which calls a few other methods